src/App.tsx

```diff
@@ -1,24 +1,310 @@
-      <p className="title is-4">
-        Copy all you need from the prev task:
-        <br />
-        <a href="https://github.com/mate-academy/react_todo-app-add-and-delete#react-todo-app-add-and-delete">React Todo App - Add and Delete</a>
-      </p>
-
-      <p className="subtitle">Styles are already copied</p>
-    </section>
+    <div className="todoapp">
+      <h1 className="todoapp__title">todos</h1>
+
+      <div className="todoapp__content">
+        <header className="todoapp__header">
+          <button
+            type="button"
+            className={cn('todoapp__toggle-all', {
+              active: allTodos.length !== completedTodos.length,
+            })}
+            data-cy="ToggleAllButton"
+            disabled={!allTodos.length}
+            onClick={handleToogleAll}
+          />
+
+          <form>
+            <input
+              data-cy="NewTodoField"
+              type="text"
+              className="todoapp__new-todo"
+              placeholder="What needs to be done?"
+              value={todoTitle}
+              onChange={handleChange}
+              onKeyDown={handleKeyDown}
+              ref={inputRef}
+              disabled={inputDisabled}
+            />
+          </form>
+        </header>
+
+        <TodoList todos={visibleTodos} tempTodo={tempTodo} />
+
+        {!!allTodos.length && (
+          <footer className="todoapp__footer" data-cy="Footer">
+            <span className="todo-count" data-cy="TodosCounter">
+              {itemsLeft}
+            </span>
+
+            <nav className="filter" data-cy="Filter">
+              <a
+                href="#/"
+                data-cy="FilterLinkAll"
+                className={cn('filter__link', {
+                  selected: visibleTodosType === TodosType.all,
+                })}
+                onClick={setAllTodosVisible}
+              >
+                All
+              </a>
+
+              <a
+                href="#/active"
+                className={cn('filter__link', {
+                  selected: visibleTodosType === TodosType.active,
+                })}
+                data-cy="FilterLinkActive"
+                onClick={setActiveTodosVisible}
+              >
+                Active
+              </a>
+
+              <a
+                href="#/completed"
+                className={cn('filter__link', {
+                  selected: visibleTodosType === TodosType.completed,
+                })}
+                onClick={setComplitedTodosVisible}
+                data-cy="FilterLinkCompleted"
+              >
+                Completed
+              </a>
+            </nav>
+
+            {!!completedTodos.length && (
+              <button
+                type="button"
+                className="todoapp__clear-completed"
+                data-cy="ClearCompletedButton"
+                onClick={destroyCompletedTodos}
+              >
+                Clear completed
+              </button>
```

Comment:
```suggestion
  
              <button
                type="button"
                className="todoapp__clear-completed"
                data-cy="ClearCompletedButton"
                onClick={destroyCompletedTodos}
                disabled={!completedTodos.length}
              >
                Clear completed
              </button>
```

it should help to pass one failed test. just manage disabled attribute to show or not clear completed button.
![Screenshot from 2024-02-10 18-31-24](https://github.com/mate-academy/react_todo-app-with-api/assets/33029864/a8faf9ca-736a-4c5a-ab95-eb613ef41847)